File: backend/Models/Book.cs
using System;
using System.Collections.Generic;

namespace backend.Models
{
    public partial class Book
    {
        public int BookId { get; set; }
        public string? Title { get; set; }
        public string? Description { get; set; }
        public int? Pages { get; set; }
        public int EditorialId { get; set; }
    }
}

File: frontend/src/app/models/book.ts
export class Book {
    bookId?: number;
    title: string;
    description: string;
    pages: number;
    editorialId: number;

    constructor(bookId: number, title: string, description: string, pages: number, editorialId: number) {
        this.bookId = bookId;
        this.description = description;
        this.pages = pages;
        this.editorialId = editorialId;
        this.title = title;
    }
}
